packages/elcollectooorr/skills/elcollectooorr_abci/behaviours.py

```diff
@@ -1629,7 +1633,9 @@ class ElCollectooorrFullRoundBehaviour(AbstractRoundBehaviour):
         *PostFractionsPayoutRoundBehaviour.behaviours,
         *TransferNFTAbciBehaviour.behaviours,
         *ResyncAbciBehaviour.behaviours,
+        *TerminationAbciBehaviours.behaviours,
     }
```

Comment:
Why is this needed here? I thought the with_termination does this

Reply:
`with_termination` is not implemented yet, also that would take care of the AbciApp part only. Although, I really don't like that extra effort is required on this side as well. I would suggest we make a `with_termination` decorator that is applied to the behaviour. The decorator would set both the AbciApp and Behaviour params.

Reply:
Please open an issue. I agree we should have a singular place where we add the background stuff.